templates/part.taskbody.php

```diff
@@ -35,6 +35,12 @@
             title="<?php p($l->t('Toggle subtasks')); ?>">
         </span>
     </a>
+    <a class="handler" ng-click="toggleChecklist(task)">
```

Comment:
If this is per task, we should use a different icon than for the subtasks toggling.